src/components/Sections/SectionHeader.tsx

```diff
@@ -8,45 +8,68 @@ import DotsIcon from 'cozy-ui/transpiled/react/Icons/Dots'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import flag from 'cozy-flags'
 
-import { SectionHeaderProps } from 'components/Sections/SectionsTypes'
+import {
+  GroupMode,
+  SectionHeaderProps
+} from 'components/Sections/SectionsTypes'
 import { actions } from 'components/Sections/SectionActions'
+import { computeGroupMode } from './utils'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
 
 export const SectionHeader = ({
   section,
   anchorRef,
   toggleMenu,
-  menuState
-}: SectionHeaderProps): JSX.Element => (
-  <>
-    <div className="u-flex u-w-100 u-flex-justify-between">
-      <Divider className="u-mv-0 u-flex-grow-1" variant="subtitle2">
-        {section.name}
-      </Divider>
+  menuState,
+  name
```

Comment:
Seems like the `name` props was temporary and you are not using it anymore (no `<SectionHeader>` called with it). Maybe you can remove it?

Reply:
Removed it